src/llmcompressor/modifiers/quantization/gptq/base.py

```diff
@@ -93,7 +93,7 @@ class GPTQModifier(Modifier):
         and activation 8 bit quantization on the Linear layers.
     """
 
-    sequential_update: Optional[bool] = False
+    sequential_update: Optional[bool] = True
```

Comment:
nit: Should we update the doc string above, that the default for this argument is True?